### PR TITLE
added tracking links to tips

### LINKED ISSUE #5689: [CAL-453] add shortlinks to tips.tsx
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-453](https://linear.app/calcom/issue/CAL-453/add-shortlinks-to-tipstsx)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
pretty self-explanatory.

tracking links via dub.sh for tips

### CODE REVIEW

File: packages/ui/v2/modules/tips/Tips.tsx [hunk 1]
```diff
@@ -6,47 +6,55 @@ import { localStorage } from "@calcom/lib/webstorage";
 
 import Card from "../../core/Card";
 
-const tips = [
+export const tips = [
```
Comment (PeerRich) [inline packages/ui/v2/modules/tips/Tips.tsx:9]: exporting so i can use it on cal.com/videos soon

**Review by alannnc — APPROVED** [review]
Looks good! Every link works

**Review by zomars — APPROVED** [review]

File: packages/ui/v2/modules/tips/Tips.tsx [hunk 2]
```diff
@@ -6,47 +6,55 @@ import { localStorage } from "@calcom/lib/webstorage";
...
     description: "Charge for your time with Cal.com's Stripe App",
     href: "https://app.cal.com/apps/stripe",
   },
+  {
+    id: 6,
+    thumbnailUrl: "https://img.youtube.com/vi/yGiZo1Ry5-8/0.jpg",
+    mediaLink: "https://go.cal.com/recurring-video",
+    title: "Recurring Bookings",
+    description: "Learn how to create a recurring schedule",
+    href: "https://go.cal.com/recurring-video",
```
Comment (emrysal) [inline packages/ui/v2/modules/tips/Tips.tsx:56]: Is this `href` correct (seeing all others have non-go. href's)?

**Review by emrysal — APPROVED** [review]

File: packages/ui/v2/modules/tips/Tips.tsx (hunk 2 above)
Comment (PeerRich) [reply packages/ui/v2/modules/tips/Tips.tsx:56]: yes

**Pull request merged**